scripts/paas_app_wrapper.sh

```diff
@@ -30,7 +30,7 @@ case $NOTIFY_APP_NAME in
     ;;
   delivery-worker-reporting)
     exec scripts/run_app_paas.sh celery -A run_celery.notify_celery worker --loglevel=INFO --concurrency=11 \
-    -Q reporting-tasks 2> /dev/null
+    -Q reporting-tasks -Ofair 2> /dev/null
```

Comment:
If we set prefetch multiplier to 1 do we need the -Ofair option?

Reply:
Yep, The prefetch multiplier is how many items to prefetch from SQS.

The `-Ofair` option is about once those items have been prefetched from SQS and are being held by the main master celery master process, does it automatically assign them to a celery child process or does it hold on to them and wait for a child process to free itself and then give the task to the child process.

They are both about "prefetching" but the first is from the broker and the second is from the master celery process. They solve different problems but are both beneficial.